### DISCUSSION TITLE
Digitization and Geosamples

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#5399] [Announcements]
@Nicole-Ridgwell-NMMNHS @cemyers42 @beth3ha

I chatted with [Adam Mansur](https://naturalhistory.si.edu/staff/adam-mansur) from the Smithsonian today at Biodiversity Digitization: Celebrating a decade of progress. His poster is [here](https://drive.google.com/file/d/1b_kP4mncL9agmE_fPd6r_zeDzAcP_IiH/view?usp=sharing) and a recording for it is [here](https://drive.google.com/file/d/1CefvMGQc60rJcefznW569JGZgFiFX7hc/view).

Just wanted to let you know about his work and hopefully connect you to the community that will be doing for geosamples what GBIF etc. are doing for biosamples.

One thing he mentioned that sounded cool was [Earth Science Information Partners](https://www.esipfed.org/) and from his poster [GeoCase](https://geocase.eu/) where maybe we should be publishing your data?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I've heard of GeoCase before, but looks it just got upgraded. It would be good to explore, though we'd have to map the data over to ABCD-EFG. I was trying to see if they have any North American collections in there, looks like it is still mostly Europe.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
There were some interesting updates relevant to this in the TDWG ESP interest group today. 
- Mindat is getting an overhaul and an api ([https://www.mindat.org/mesg-567769.html](https://www.mindat.org/mesg-567769.html)) 
- Ben Norton at the North Carolina Museum of Natural Science is working on forming a TDWG task group for data standards for mineral, rock and meteorite collections.
- GeoCase might be able to take Darwin Core???

**Comment by Jegelewicz** [comment]
@Nicole-Ridgwell-NMMNHS are you going to join that task group? It would be great if you could!!!

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I'll try to at least listen in keep up to date on what they are doing.

**Comment by Jegelewicz** [comment]
@beth3ha this might interest you guys as well.
